lib/blacklight/models/staff_info.rb

```diff
@@ -0,0 +1,71 @@
+
+module Blacklight
+  class StaffInfo
+    attr_accessor(
```

Comment:
Jamis was telling me that unless you need to edit the values on the class from another file then have this be attr_reader. In looking at the code, I don't know if you are accessing very many of these except in your specs. And then to access private variables you can do something like (@question.instance_variable_get :@max_score).